services/github/github-downloads.service.js

```diff
@@ -36,11 +41,10 @@ module.exports = class GithubDownloads extends LegacyService {
           user: 'atom',
           repo: 'atom',
         },
```

Comment:
any objections to adding `download` as a keyword? Happy to push a commit myself if there's no objections.

I know it wasn't in there before, but I just noticed it when I went into the staging app and tried to search for `github downloads`

Reply:
Yes please!

Reply:
done